docs/client/rest/rest_api.md

```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
 |:-----------|:----------------------------|:-------|
 | identifier | The identifier of operation | String |
 
+## Operation Resource
+
+### GET /operations/${operationHandle}/event
+
+Get an event of operation
+
+#### Response Body
+
+The [KyuubiOperationEvent](#kyuubioperationevent).
+
+### PUT /operations/${operationHandle}
+
+Put an action to the operation
+
+#### Request Body
+
+| Name   | Description                                                                 | Type   |
+|:-------|:----------------------------------------------------------------------------|:-------|
+| action | The action that is applied the operation, for example: 'cancel' and 'close' | String |
+
+### GET /operations/${operationHandle}/resultsetmetadata
+
+Get a result set of metadata 
+
+#### Response Body
+
+| Name    | Description                 | Type               |
+|:--------|:----------------------------|:-------------------|
+| columns | The descriptions of columns | List of ColumnDesc |
+
+### GET /operations/${operationHandle}/log
+
+Get a set of operation logs
```

Comment:
Get a list of operation log lines of the running operation by the specified operation handle.